### DISCUSSION TITLE
sorry!when can we use this project as a golang library?

### DISCUSSION THREAD

**Discussion by Nyx2022** [projectdiscovery/httpx#339] [General] [answered]
**As this title**
I wanna know that when can we use httpx as project subfinder's EnumerateSingleDomain method in our project?maybe i'm too anxious

**Comment by Mzack9999** [comment, marked as answer]
@Nyx2020 All packages are exported, so indeed, it's possible to use httpx as a library. Anyway, httpx is not meant to enumerate subdomains. For that, I think you should look into subfinder.

**Reply by Nyx2022** [reply]
i mean i wanna give a input like []string to httpx and receive a result like []Result,
so can realise this goal?
